controllers/servers/utils.py

```diff
@@ -606,11 +606,16 @@ def generate_addons_replication_request(request, replication_type):
 
     other_system_id = request.parameters.get(servers_settings.PARAMETERS_SYSTEM_ID)
     copy_type = request.parameters.get(servers_settings.PARAMETERS_COPY_TYPE, REPLICATION_DEFAULT_COPY_TYPE)
+
+    replication_policy = None
+    if servers_settings.PARAMETERS_REPLICATION_POLICY in request.parameters:
+        replication_policy = request.parameters.get(servers_settings.PARAMETERS_REPLICATION_POLICY)
```

Comment:
if you use `.get` I don't think you need the `if` or the  initialization

Reply:
Done